Documentation/gettingstarted/egress-gateway.rst

```diff
@@ -385,12 +385,36 @@ cluster with the IP of the node.
 Apply egress gateway policy
 ---------------------------
 
+Download the ``egress-sample`` Egress Gateway Policy yaml:
+
+.. code-block:: shell-session
```

Comment:
I _think_ this needs to be a `parsed-literal::` in order to substitute `|SCM_WEB|` with the full path

Reply:
Correct